Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM python:3.11-slim AS minimal
 
 ENV VERSION_CT=0.9.0 \
-    VERSION_HELM=3.16.4
+    VERSION_HELM=3.16.4 \
```

Comment:
helm is already at v3.18.4, any specific reason why we still use v3.16 ?

Reply:
I raised a dedicated PR for updates in #44. This was only for shipping the new module.